f2/src/utils/nextWhatWasAffectedUrl.js

```diff
@@ -1,16 +1,114 @@
-import { whatWasAffectedPages } from '../forms/WhatWasAffectedForm'
-
-export const nextWhatWasAffectedUrl = (optionsSelected, currentPage) => {
-  let pages = whatWasAffectedPages.filter(
-    page => page.url !== '' && optionsSelected.indexOf(page.key) > -1,
-  )
-  pages.unshift({ key: '', url: 'whatwasaffected' }) // start of list
-  pages.push({ key: '', url: 'whathappened' }) // end of list
-
-  for (const [index, page] of pages.entries()) {
-    if (page.url === currentPage) {
-      return pages[Math.min(index + 1, pages.length)].url
+const { getLogger } = require('./winstonLogger')
+
+const logger = getLogger(__filename)
+
+export const whatWasAffectedPages = {
+  FINANCIAL: {
+    key: 'whatWasAffectedForm.financial',
+    url: 'moneylost',
+  },
+  INFORMATION: {
+    key: 'whatWasAffectedForm.personalInformation',
+    url: 'information',
+  },
+  DEVICES: {
+    key: 'whatWasAffectedForm.devices',
+    url: 'devices',
+  },
+  BUSINESS: {
+    key: 'whatWasAffectedForm.business_assets',
+    url: 'business',
+  },
+  OTHER: {
+    key: 'whatWasAffectedForm.other',
+    url: '',
+  },
+  FIRST_PAGE: {
+    key: '',
+    url: 'whatwasaffected',
+  },
+  LAST_PAGE: {
+    key: '',
+    url: 'whathappened',
+  },
+  CONFIRMATION: {
+    key: '',
+    url: 'confirmation',
+  },
+}
+
+export const pages = {
```

Comment:
This would be much easier to understand if we call it whatWasAffectedNavState or some sort. pages creates lots of confusion when I read through the code especially at those selected pages.

Reply:
Object has been renamed.